poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
+            </div>
+            {{#if any_img}}
+                <div class="poll-image result-image">
+                    <label for="answer-{{key}}" class="poll-image-label">
+                        {{#if img}}
+                          <img src="{{img}}" />
+                        {{/if}}
+                    </label>
+                </div>
+            {{/if}}
+            <div class="percentage-gauge-container">
+              <div class="percentage-gauge" style="width:{{percent}}%;">
+                  <label class="poll-answer-label" for="answer-{{key}}">{{{answer}}}</label>
```

Comment:
@smarnach I don't think paragraph elements in a <label> are the end of the world, if we knew for sure that would be the worst thing that was ever put in there.  When I think of everything we can do with Markdown, I get a little nervous.  Multiple paragraphs would be a problem.  Links would be a problem (even though they qualify as phrasing content).  

Personally, I wouldn't write invalid HTML, but if you can sleep at night, I'm OK with it ;).  I'm convinced that this is something we can address with documentation (by highlighting the features of Markdown that are allowed here, really just italics and bold).